src/model_theory/order.lean

```diff
@@ -89,137 +89,145 @@ Lhom.funext (subsingleton.elim _ _) (subsingleton.elim _ _)
 
 instance : is_ordered (L.sum language.order) := ⟨sum.inr is_ordered.le_symb⟩
 
+section
+variables (L) [is_ordered L]
+
 /-- The theory of preorders. -/
-protected def Theory.preorder : language.order.Theory :=
+def preorder_theory : L.Theory :=
```

Comment:
IIUC this name change is motivated by dot notation acting up when you try to make `language.order.Theory.preorder` mean `Theory.preorder language.order`, right? Can you record that (or whatever is actually true) in the PR description?

Reply:
I've added a clarifying comment in the PR description - once this definition is language-dependent, it should be directly in the language namespace, so `Theory.` should not be part of the name.

Reply:
Can you add a more explicit example in the PR description comparing the before/after of one of the fully-qualified names, and gives an `example` that motivates the change?

I trust that this change is probably reasonable, but it's hard for me to tell what is going on from the PR description.